mbed-os.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#64853b354fa188bfe8dbd51e78771213c7ed37f7
+https://github.com/ARMmbed/mbed-os/
```

Comment:
We do not want it to point at any specific version?

Reply:
I think that master should be the latest development branch like in mbed-os. Many examples (not all) follow that pattern.